internal/core/services/scheduler_manager/scheduler_manager_test.go

```diff
@@ -46,6 +46,68 @@ import (
 	"github.com/topfreegames/maestro/internal/validations"
 )
 
+func TestCreateScheduler(t *testing.T) {
```

Comment:
@lyracampos @caiordjesus There is no new behaviour in scheduler_manager, here I just added missing test cases and scenarios to improve coverage of scheduler manager to 100%